test/blocks/bulk-publish-v2/bulk-publish-v2.test.js

```diff
@@ -114,6 +114,16 @@ describe('Bulk Publish Tool', () => {
     await mouseEvent(rootEl.querySelector('.fix-btn'));
   });
 
+  it('can trigger cannot publish config', async () => {
+    await clock.runAllAsync();
```

Comment:
I think the `await clock.runAllAsync()` might be redundant since we do not trigger any async operation before it.

Reply:
Because Bulk Publish testing involves rendering multiple web components, these functions are serving as a timing delay in order for the components to finish processing and rendering in the tests prior assertions. Originally I had setup a timeout function to serve this purpose, but they were updated to use this `runAllAsync` function a few months ago. I did notice some test failures due to that update and fixed them in this PR which is why you can see some of the assertions are moved around. 

Not sure if there is a better solution, but for now this method is allowing the UI to finish its work before running new assertions and subsequently failing because the components have not finished rendering.